.eslintrc

```diff
@@ -43,7 +43,7 @@
       "never"
     ],
     "one-var": [
-      2,
+      0,
```

Comment:
Moreover, the rule exists to help avoid bugs that creep in because of hoisting of `var` statements, that's not true anymore in the codebase.